src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -535,7 +535,7 @@ private static void CheckBaseAddress(Uri baseAddress, string parameterName)
 
         private void SetTimeout(CancellationTokenSource cancellationTokenSource)
         {
-            Contract.Requires(cancellationTokenSource != null);
+            Debug.Assert(cancellationTokenSource != null);
```

Comment:
Removed the usings and the references that aren't either needed or are brought in by dependencies. Should be more version consistency also.